src/Directory.Build.props

```diff
@@ -15,6 +15,7 @@
     <DebugType>embedded</DebugType>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <NoWarn>$(NoWarn);1591</NoWarn> <!--TODO: temporary solution-->
```

Comment:
With this change, let's remove ` -p:NoWarn=CS1591` from the github action workflow(s).

Reply:
I expected this suggestion :)

Reply:
done